src/computer.h

```diff
@@ -67,6 +67,10 @@ enum computer_failure {
     NUM_COMPUTER_FAILURES
 };
 
+void initialize_computer_maps();
+static std::map<std::string, computer_action> string_to_action;
```

Comment:
You generally don't want to expose the actual container globally like this, you want to just expose accessors to it like computer::add_action(), computer::add_failure(), computer::get_action(), etc.